### ISSUE TITLE
Python3.8: PY_SSIZE_T_CLEAN will be required for '#' formats

### ISSUE THREAD

**Issue by openedhardware** [chemicstry/candle_driver#2]
Here is a log when I executed the testing script with python3.8:
```
test.py:17: DeprecationWarning: PY_SSIZE_T_CLEAN will be required for '#' formats
  ch.write(10, b'abcdefgh')
test.py:20: DeprecationWarning: PY_SSIZE_T_CLEAN will be required for '#' formats
  frame_type, can_id, can_data, extended, ts = ch.read(1000)
```

I think this can fix this issue? https://github.com/mongodb/winkerberos/pull/38

**Comment by chemicstry** [comment]
I currently don't have candle device on hand to test this properly and I'm not sure when I will.

If you can fix it and test it I will gladly accept a PR though

**Comment by chemicstry** [comment]
Fixed the issue in 03c6a8f8f39c2bdebc1b207748c4ad3fa06f2786
